include/linux/blkdev_compat.h

```diff
@@ -351,8 +351,8 @@ bio_set_flush(struct bio *bio)
 	bio_set_op_attrs(bio, 0, REQ_PREFLUSH);
 #else
 #error	"Allowing the build will cause bio_set_flush requests to be ignored."
-	"Please file an issue report at: "
-	"https://github.com/zfsonlinux/zfs/issues/new"
+#error	"Please file an issue report at: "
+#error	"https://github.com/zfsonlinux/zfs/issues/new"
```

Comment:
That doesn't seem to work.  Moving to a single line would work except for the last instance, unless you can suggest a shorter message.

```
 #error  "Allowing the build will cause bio_set_flush requests to be ignored." \
  ^
make: *** [Makefile:5: all] Error 1
```